script/tool/lib/src/format_command.dart

```diff
@@ -206,7 +206,27 @@ class FormatCommand extends PluginCommand {
 
     final String fromPath = relativeTo.path;
 
+    // Dart files are allowed to have a pragma to disable auto-formatting. This
+    // was added because Hixie hurts when dealing with what dartfmt does to
+    // artisanally-formatted Dart, while Stuart gets really frustrated when
+    // dealing with PRs from newer contributors who don't know how to make Dart
+    // readable. After much discussion, it was decided that files in the plugins
```

Comment:
🤣 Well, I did ask.

(I was just thinking something like "Allow package maintainers to opt out of Dart autoformatting in order to use Flutter's manual formatting style instead.", but this certainly works.)